proposals/0002-web-rtc.md

```
@@ -0,0 +1,455 @@
+        }
+    }
+}
+
+// Send/receive messages using Kotlin's `Channel`-like API. 
+scope.launch { channel.send("hello") }
+scope.launch { println("recv: " + channel.receiveText()) }
+```
+
+### Media tracks
+
+```kotlin
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)